crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/enums/staticdata/AppealContributionAmountTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.justice.laa.crime.contribution.enums.staticdata;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import uk.gov.justice.laa.crime.contribution.staticdata.enums.AppealContributionAmount;
+import uk.gov.justice.laa.crime.enums.AppealType;
+import uk.gov.justice.laa.crime.enums.AssessmentResult;
+import uk.gov.justice.laa.crime.enums.CrownCourtAppealOutcome;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class AppealContributionAmountTest {
+    @ParameterizedTest
+    @MethodSource("appealTypeAndAppealOutcomeCombinations")
+    public void givenASuccessfulAssessmentResult_thenZeroContributionAmountIsReturned(AppealType appealType, CrownCourtAppealOutcome appealOutcome) {
+        AppealContributionAmount result = AppealContributionAmount.calculate(appealType, appealOutcome, AssessmentResult.PASS);
+
+        assertThat(result).isEqualTo(AppealContributionAmount.NO_CONTRIBUTION);
+    }
+
+    @ParameterizedTest
+    @MethodSource("appealTypes")
+    public void givenAFailedAssessmentResultAndSuccessfulAppealOutcome_thenZeroContributionAmountIsReturned(AppealType appealType) {
```

Comment:
JUnit5 test classes and methods should have default package visibility.

Reply:
Thanks, will update!

Reply:
Done